rewrite-gradle/src/test/java/org/openrewrite/gradle/UpgradeDependencyVersionTest.java

```diff
@@ -394,6 +395,99 @@ implementation platform("com.google.guava:guava:30.1.1-jre")
         );
     }
 
+    @Issue("https://github.com/openrewrite/rewrite/issues/5300")
+    @Test
+    void doesNotMakeChangesInDependencyManagement() {
```

Comment:
This test feels a bit extraneous to me since it's effectively never going to do anything from a recipe standpoint.